lib/index.js

```diff
@@ -0,0 +1,169 @@
+require("dotenv").config();
+const { OpenAI } = require("openai");
+const fs = require("fs");
+const axios = require("axios");
+const cheerio = require("cheerio");
+const pretty = require("pretty");
+
+const OUTPUT_FOLDER = "summaries";
+
+const fileSystemSetup = () => {
```

Comment:
This file is used to setup the folders necessary for our agent to perform its tasks. It returns some variables that are necessary down the line.